lib/shared/bucketing-assembly-script/assembly/managers/eventQueueManager.ts

```diff
@@ -121,6 +121,7 @@ export function queueEvent(
         _getConfigData(sdkKey),
         dvcUser,
         _getClientCustomData(sdkKey),
+        null,
```

Comment:
if this is always `null` does that mean the `feature: variation` ids will be skipped from the featureVars map for overrides / self-targeting features?

Reply:
technically yeah, but the event queue logic is only used by serverside SDKs, which currently would not deal with overrides directly. The only thing that matters is the code used by the bucketing API, which is just the config generator  portion.